cmake/std/atdm/ats1/all_supported_builds.sh

```diff
@@ -0,0 +1,13 @@
+
+export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-
+
+export ATDM_CONFIG_ALL_SUPPORTED_BUILDS=(
+  ats1-hsw_intel-19.0.4_mpich-7.7.6_openmp_static_opt
+  ats1-hsw_intel-19.0.4_mpich-7.7.6_openmp_static_dbg
+  ats1-knl_intel-19.0.4_mpich-7.7.6_openmp_static_opt
+  ats1-knl_intel-19.0.4_mpich-7.7.6_openmp_static_dbg
+  ats1-hsw_intel-18.0.5_mpich-7.7.6_openmp_static_opt
+  ats1-hsw_intel-18.0.5_mpich-7.7.6_openmp_static_dbg
+  ats1-knl_intel-18.0.5_mpich-7.7.6_openmp_static_opt
+  ats1-knl_intel-18.0.5_mpich-7.7.6_openmp_static_dbg
```

Comment:
This list should be the actual set of builds we are running in nightly testing.  There is no way we can run this many builds currently with the current jenkins-srn.sandia.gov slave (with max 2 executors).  I think we are going to need to trim down significantly or we are going to need to switch to running these with a massively parallel cron driver.  (I actually have that a driver script that can do that in the repo https://gitlab.sandia.gov/atdm/atdm-trilinos-nightly-builds.  I used that for the driver on 'stria' until they got the Jenkins slave up.)